tests/cpp/tests/test_bqm.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 D-Wave Systems Inc.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+//
+// =============================================================================
+
+#include "../Catch2/single_include/catch2/catch.hpp"
+#include "../../../dimod/include/dimod/adjvectorbqm.h"
```

Comment:
I think this should just be `#include "dimod/adjvectorbqm.h"` and we should add the c++ source directory to the include path when compiling.